network-api/networkapi/fellows/templates/fellows_apply.html

```diff
@@ -10,27 +10,53 @@
     <div class="row">
       <div class="col-12">
         <h1 class="h1-white text-capitalize">Be a fellow</h1>
-        <p>Mozilla Fellowships provide resources, tools, community and amplification to those building a more ​humane​ ​digital​ ​world. During their tenure, Fellows use their skill sets — in technology, in advocacy, in law — to lay the groundwork for a more open and inclusive internet. Mozilla Fellows hail from a range of disciplines and geographies, and are awarded competitive compensation and benefits.</p>
-        <h3 class="h5-black mb-1">Science</h3>
-        <p class="mb-4">Our science-focused fellows are researchers who want to increase the openness of data sharing of science research communities.</p>
-        <h3 class="h5-black mb-1">Open Web</h3>
-        <p class="mb-4">Our fellows working on issues to support the growing field of public interest technology are passionate about working increasing transparency, privacy and security in the open web.</p>
-        <h3 class="h5-black mb-1">Tech Policy</h3>
-        <p class="mb-4">Our fellows focused on technology policy-related issues around the world that are central to open public policy at the intersection of legal, academic, and governmental action.</p>
-        <h3 class="h5-black mb-1">Media</h3>
-        <p class="mb-4">Our fellows in open media are creators and media-makers inspired by the open web to explore the boundaries of technology privacy, politics, public awareness, and surveillance.</p>
-      </div>
-      <div class="col-12 mt-2 mb-5">
-        <a href="https://mozilla.fluxx.io/apply/2018fellows" class="btn btn-normal">Application Coming Soon</a>
-      </div>
-    </div>
+        <p>Mozilla Fellowships provide resources, tools, community and amplification to those building a more ​humane​ ​digital​ ​world. During their tenure, Fellows use their skill sets — in technology, in activism, in science, in policy — to design products, run campaigns, influence policy and ultimately lay the groundwork for a more open and inclusive internet.
+        </p>
+        <p>Who are Mozilla Fellows? They are working to stop the spread of misinformation, to put individuals in control of their data, and to keep artificial intelligence accountable. They are leaders who ensure smart cities and next-generation voice technology are diverse and equitable, and who conduct open research.
+        </p>
+        <p>Mozilla Fellowships are a transformative experience for emerging leaders concerned with making the internet a safer, more accessible resource for everyone. Fellows expand their network and sphere of influence; design impactful projects with the potential to reach millions; and learn from and collaborate with a global community of thousands of Mozillians. Mozilla Fellows are also awarded competitive compensation and benefits. 
+        </p>
+        <h4 class="h4-light-black">Mozilla Fellows fit three particular profiles:</h4>
+      </div><!-- end of col 12 -->
```

Comment:
Please don't add all these extra `end/beginning` comments. This is just extra noise for editors with code folding (Atom, Sublime, etc) and more to maintain to retain accuracy if the class names change.

Reply:
Sure! Thanks for the tip Gavin! I will change it and submit the PR again I guess?